image/templates/helm/stackrox-central/templates/NOTES.txt.htpl

```diff
@@ -3,6 +3,20 @@
 StackRox {{.Chart.AppVersion}} has been installed.
 
 [< if not .Operator >]
+Central Services Configuration Summary:
+
+  Name:                                        {{ ._rox.clusterName }}
+  Kubernetes Namespace:                        {{ ._rox._namespace }}{{ if ne .Release.Namespace ._rox._namespace }}[NOTE: Helm release is attached to namespace {{ .Release.Namespace }}]{{ end }}
+  Helm Release Name:                           {{ .Release.Name }}
+  Central Endpoint:                            {{ ._rox.centralEndpoint }}
+  OpenShift Cluster:                           {{ if eq ._rox.env.openshift 0 -}} false {{ else -}} {{ ._rox.env.openshift }} {{ end }}
+[<- if .FeatureFlags.ROX_SCANNER_V4 >]
+  Scanner v4 Mode:                             {{ ._rox.scannerV4._installMode }}
```

Comment:
Nitpick: Can we use "Scanner V4 Installation Method" instead of "Mode" to not get confused about the Scanner mode of operation we have in the Scanner Configuration YAML?